app/frontend/src/components/Markdown.tsx

```diff
@@ -17,10 +17,12 @@ const useStyles = makeStyles((theme) => ({
     fontFamily: theme.typography.fontFamily,
     "& h1": {
```

Comment:
This should probably be have the same `"& h1, & h2, & h3, & h4, & h5, & h6"`as the other one